15/umbraco-forms/developer/configuration/README.md

```diff
@@ -446,6 +447,15 @@ This setting determines whether [multi-page form settings](../../editor/creating
 
 By default the value is `true`. To disable the feature, set the value to `false`.
 
+## EnableAdvancedValidationRules
+
+This setting determines whether [advanced form validation rules](../../editor/creating-a-form/form-advanced.md) are available to editors.
+
+By default the value is `false`.  This partly because the feature is considered something only for "power users", comfortable with crafting rules using the JSON syntax required. And partly as validating the rules on the client requires an additional front-end dependency.
```

Comment:
```suggestion
By default, the value is `false`.  This is partly because the feature is only considered for "power users", comfortable with crafting rules using the required JSON syntax. And partly as validating the rules on the client requires an additional front-end dependency.
```